Sources/Stagehand/AnimationQueue.swift

```diff
@@ -1,5 +1,6 @@
 //
-//  Copyright 2019 Square Inc.
+//  Portions of this file are Copyright 2025 Nick Entin
+//  Portions of this file are Copyright 2019 Square Inc.
```

Comment:
Is there prior art for this? Not sure how this works tbh

Reply:
Found a lot of conflicting stuff on how you're supposed to write this when you fork. Seems like some variant of this is probably right, but not totally clear. I guess I'll just leave it as-was for now.
```suggestion
//  Copyright 2019 Square Inc.
```